en: add crd-based backup and restore docs (#1)
pingcap/docs-tidb-operator: en/backup-and-restore-using-helm-charts.md

---
title: Helm Charts-Based Backup and Restoration
summary: Learn how to back up and restore the data of TiDB cluster in Kubernetes based on Helm charts.
category: how-to
aliases: ['/docs/dev/tidb-in-kubernetes/maintain/backup-and-restore/','/docs/v3.0/tidb-in-kubernetes/maintain/backup-and-restore/','/docs/v3.1/tidb-in-kubernetes/maintain/backup-and-restore/']
---

# Helm Charts-Based Backup and Restoration

This document describes how to back up and restore the data of a TiDB cluster in Kubernetes based on Helm charts.

For TiDB Operator 1.1 or later versions, it is recommended that you use the backup and restoration methods based on CustomResourceDefinition (CRD). Refer to the following documents for details:

* [Back up Data to GCS](backup-to-gcs.md)
* [Restore Data from GCS](restore-from-gcs.md)
* [Back up Data to S3-Compatible Storage](backup-to-s3.md)
* [Restore Data from S3-Compatible Storage](restore-from-s3.md)

TiDB in Kubernetes supports two kinds of backup strategies (based on Helm charts):

* [Full backup](#full-backup) (scheduled or ad-hoc): use [`mydumper`](https://pingcap.com/docs/stable/reference/tools/mydumper) to take a logical backup of the TiDB cluster.

> * [Incremental backup](#incremental-backup): use [`TiDB Binlog`](https://pingcap.com/docs/stable/reference/tidb-binlog/overview) to replicate data in the TiDB cluster to another database or take a real-time backup of the data.

Reviewer: ```suggestion
* [Incremental backup](#incremental-backup): use [TiDB Binlog](https://pingcap.com/docs/stable/reference/tidb-binlog/overview) to replicate data from the TiDB cluster to another database or execute a real-time backup of the data.
```
